### PR TITLE
Components

### PR DESCRIPTION
**Pull request by safafa**
In this PR, I developed a React component that holds the core functionality: a calculator.
- Created a class component called calculator;
- Imported the component in the App component;
- Added some style to match the given design using CSS.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
-    <div className="App">
```
Comment (henatan99) [inline src/App.js]: - [x] The App.js component should be class-based as per the project's requirement. In this milestone, the objective is to practice the class-based React component. :pray:

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
-  height: 40vmin;
-  pointer-events: none;
+.screen {
+  background-color: #858694;
+  text-align: right;
+  margin-bottom: 0;
+  height: 3rem;
+  color: white;
+  font-weight: bold;
 }
```
Comment (henatan99) [inline src/App.css]: - [x] The text in the screen element is not positioned properly. Kindly compare it with the original design and make changes to your styling. :pray: 

image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @safafa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] You can make your PR summary more descriptive by listing out briefly the main changes made in this branch. :pray: e.g:- Implemented the Calculator component. 
- [x] Kindly update your readme with a minimalist description about the project in the about section. :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @safafa, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️


- [x] As said by the previous reviewer, Kindly update your readme with a minimalist description about the project. :pray:. Currently, you have no project description in the README file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (henatan99) [reply src/App.js]: This issue is well addressed. Great work! :clap:

File: src/App.css (hunk 2 above)
Comment (henatan99) [reply src/App.css]: This issue is resolved. Job well done! :clap: :+1:

**Review by henatan99 — APPROVED** [review]
Hi @safafa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**